CMakeLists.txt

```diff
@@ -120,11 +121,15 @@ include_directories(${SUBSTRAITCPP_INCLUDE_DIR})
 #
 # Subdirectories
 #
-include_directories(cider)
-include_directories(cider/include)
-add_subdirectory(cider)
+include_directories(src/cider)
```

Comment:
if we add some other language bind(which doesn't use cmake), then we should change `src` to `cpp` and delete top-level `CMakeLists.txt`

Reply:
if add other language, the folder structure likes
```
├── cpp
│   ├── CMakeLists.txt
│   └── src
│       ├── cider
│       ├── cider-velox
│       └── compression
└── java
    ├── cider
    ├── compression
    └── pom.xml

```

Reply:
OK